lizmap/modules/lizmap/controllers/searchFts.classic.php

```diff
@@ -30,12 +30,58 @@ public function get()
         if (!$pquery) {
             return $rep;
         }
+
+        $repository = $this->param('repository');
+        if (!$repository) {
+            // The parameter repository is mandatory !
+
+            return $rep;
+        }
+
         $project = $this->param('project');
+        if (!$project) {
+            // The parameter project is mandatory !
+
+            return $rep;
+        }
+
+        // Check repository
+        if (!lizmap::getRepository($repository)) {
+            jLog::log('The repository '.strtoupper($repository).' does not exist !', 'errors');
+
+            return $rep;
+        }
+
+        // Get the project object
+        $lproj = null;
+
+        try {
+            $lproj = lizmap::getProject($repository.'~'.$project);
+            if (!$lproj) {
+                jLog::log('The lizmap project '.strtoupper($project).' does not exist !', 'errors');
+
+                return $rep;
+            }
+        } catch (UnknownLizmapProjectException $e) {
+            jLog::logEx($e, 'error');
+            jLog::log('The lizmap project '.strtoupper($project).' does not exist !', 'errors');
+
+            return $rep;
+        }
+
+        // Redirect if no rights to access this repository
+        if (!$lproj->checkAcl()) {
+            //jMessage::add(jLocale::get('view~default.repository.access.denied'), 'AuthorizationRequired');
+
+            return $rep;
+        }
+
         $pquery = filter_var($pquery, FILTER_SANITIZE_STRING, FILTER_FLAG_NO_ENCODE_QUOTES);
 
         // Run query
         $fts = jClasses::getService('lizmap~lizmapFts');
-        $data = $fts->getData($project, $pquery);
+        jLog::log('The lizmap project '.strtoupper($lproj->getKey()).' exists!', 'errors');
```

Comment:
error ? while the project exist ?
Maybe remove this log.

Reply:
it was for testing, something wrong in the docker refreshing files